cypress/integration/advanced-functionalities/admin/system/messaging-queues/Edit.spec.js

```diff
@@ -0,0 +1,32 @@
+/// <reference types="cypress" />
+const adminURL = Cypress.env('ADMIN_URL')
+const email = Cypress.env('USER_EMAIL')
+const password = Cypress.env('USER_PASSWORD')
+
+describe('Test for editing a messaging queue', () => {
+  before(() => {
+    if (!window.sessionStorage.getItem('auth.refresh-token')) {
+      cy.login({ email, password, url: adminURL })
+    }
+  })
+
+  context('Test for editing a messaging queue', () => {
+    it('should be able to edit a messaging queue', () => {
+      cy.visit(adminURL + '/')
+      // We wait 3s in order the page to be fully loaded
+      cy.wait(3000)
+      cy.get('.nav-sidebar').contains('Messaging Queues').click()
+      // We wait 1s in order the page to be fully loaded
+      cy.wait(1000)
+      cy.get('[data-test-id="input-search"]').type('TestQueue')
+      cy.get('#resource-list > tbody > tr:last > td:last > a').click()
+      cy.get('[data-test-id="input-name"]').type('Edited')
+      cy.get('[data-test-id="input-consumer"]').select('Corteza')
```

Comment:
Create a msg queue with the two other consumer options - redis and eventbus. Save the msg queue and test if everything is okay